### ISSUE TITLE
Command-block problem

### ISSUE THREAD

**Issue by Mrconium** [Minehut/Issues#181]
### Expected behavior
[answer]: I expect that Minehut staff add /minehut command in lobby and not only in the server

### Actual behavior
[answer]: behavior? 

### Detailed steps to reproduce this behavior
[answer] the problem is that anyone op'ed can place a repeating command block with /kill @a and there is no way to fix this than deleting our server becuz we cant even make command block un-enabled anymore. This is quite harrasing because i lose my whole server becuz i gave /op to someone and allow them to use command blocks but most of the servers do need command blocks to work so it is a pain to find someone trustworthy that can become a dev for you and that does know how cmd blocks work and that doesnt greif the server. Now that my server has been *attacked* i have no other choice then deleting it unless MH staff has access to the console and can deactivate cmd blocks. I Will leave my server like that for a few days hoping MH staff can disable cmd blocks. If the /minehut cmd was enabled from the lobby, that could of been fixed by myself already. I really hope you guys can do something about it. Note: my server is whitelisted so you might need to change stuff to enter in it and disable cmd blocks.

### Server name
[answer]: WhereIsTheButton

### Link to server logs (/dl log)
[answer] srry idk what that is

**Comment by tjhubz** [comment]
The simple fix to this is to not grant elevated permissions to players you don't fully know. While this feature may be added in the future, it most likely will not be an urgent change.

**Comment by GalaxyShadow** [comment]
Agreed. I am very careful with ranks, and even the highest staff ranks do not have permissions for command blocks. Be careful!
